### PR TITLE
Initial review of account.rst:

### PR DESCRIPTION
**Pull request by Mrt134**
Update account.rst to match the options in the new UI.
Adjust links to use two underscores.
Clean tables up a little.
HTML build test: issues with tasks.rst that need to be resolved separately (not related to this work).

### CODE REVIEW

File: userguide/account.rst [hunk 1]
```diff
@@ -48,11 +48,11 @@ to see a screen like
 
 
 All groups that came with the operating system will be listed. Each
```
Comment (wblock) [inline userguide/account.rst]: Fix this.

File: userguide/account.rst [hunk 2]
```diff
@@ -83,31 +83,31 @@ summarizes the available options when creating a group.
    |                     |           |                                                                                                                          |
    |                     |           |                                                                                                                          |
    +=====================+===========+==========================================================================================================================+
-   | Group ID            | string    | the next available group ID will be suggested for you; by convention, UNIX groups containing user accounts have          |
-   |                     |           | an ID greater than 1000 and groups required by a service have an ID equal to the default port number used by the         |
-   |                     |           | service (e.g. the sshd group has an ID of 22)                                                                            |
+   | GID                 | string    | The next available group ID is suggested. By convention, UNIX groups containing user accounts have an ID greater than    |
```
Comment (wblock) [inline userguide/account.rst]: Why the added bold on these numbers?

File: userguide/account.rst [hunk 3]
```diff
@@ -83,31 +83,31 @@ summarizes the available options when creating a group.
...
+   | Name                | string    | Enter a name for the new group. This is required.                                                                        |
    |                     |           |                                                                                                                          |
    +---------------------+-----------+--------------------------------------------------------------------------------------------------------------------------+
-   | Permit Sudo         | checkbox  | if checked, members of the group have permission to use `sudo <http://www.sudo.ws/>`_; when using sudo, a user will      |
-   |                     |           | be prompted for their own password                                                                                       |
+   | Permit Sudo         | checkbox  | Check this to allow members of the group to use `sudo <http://www.sudo.ws/>`__. When using :command:`sudo`, a user is    |
+   |                     |           | prompted for their own password.                                                                                         |
```
Comment (wblock) [inline userguide/account.rst]: Looks like trailing whitespace added here.

File: userguide/account.rst [hunk 4]
```diff
@@ -83,31 +83,31 @@ summarizes the available options when creating a group.
...
    |                     |           |                                                                                                                          |
    +---------------------+-----------+--------------------------------------------------------------------------------------------------------------------------+
-   | Allow repeated GIDs | checkbox  | allows multiple groups to share the same group id (GID); this is useful when a GID is already associated with the        |
-   |                     |           | UNIX permissions for existing data                                                                                       |
+   | Allow repeated GIDs | checkbox  | Check this to allow multiple groups to share the same group id (GID). This is useful when a GID is already associated    |
```
Comment (wblock) [inline userguide/account.rst]: Don't say "Check this", just say "Check".

File: userguide/account.rst [hunk 5]
```diff
@@ -83,31 +83,31 @@ summarizes the available options when creating a group.
...
 
 
 After a group and users are created, users can be made members
-of a group. Highlight the group where users will be assigned, then
-click the :guilabel:`Members` button. Highlight the user in the
-:guilabel:`Member users` list (which shows all user accounts on the
-system) and click :guilabel:`>>` to move that user to the right frame.
-The user accounts which appear in the right frame are added as members
-of the group.
+of a group. Highlight the group where users will be assigned and click
```
Comment (wblock) [inline userguide/account.rst]: s/made members of/added to/

File: userguide/account.rst [hunk 6]
```diff
@@ -83,31 +83,31 @@ summarizes the available options when creating a group.
...
+the :guilabel:`More Options` then :guilabel:`Members` buttons. Click the
+desired users in the :guilabel:`Members` list. This list shows all user
+accounts on the system. Next, click :guilabel:`>` to move those users to
+the right frame. The user accounts which appear in the right frame are
+added as members of the group.
```
Comment (wblock) [inline userguide/account.rst]: Added as members immediately?  Or when something else is clicked?

File: userguide/account.rst [hunk 7]
```diff
@@ -141,9 +140,9 @@ Users
 %brand% supports users, groups, and permissions, allowing great
 flexibility in configuring which users have access to the data stored
 on %brand%. To assign permissions to shares,
-**one of the following** must be done:
+**one of these options** must be done:
```
Comment (wblock) [inline userguide/account.rst]: This bold is not needed.

File: userguide/account.rst [hunk 8]
```diff
@@ -141,9 +140,9 @@ Users
...
 
-#.  Create a guest account that all users will use or create a user
+#.  Create a guest account for all users to use or create a user
```
Comment (wblock) [inline userguide/account.rst]: Needs a comma after "use".

File: userguide/account.rst [hunk 9]
```diff
@@ -168,46 +166,33 @@ operating system, as shown in
...
-built-in user that came with the %brand% installation, the email
-address, whether logins are disabled, whether the user
-account is locked, whether the user is allowed to use
-:command:`sudo`, and if the user connects from a Windows
-8 or higher system. To reorder the list, click the desired column
-name. An arrow indicates which column controls the view sort order.
-Click the arrow to reverse the sort order.
+Each account entry indicates the username, user ID, primary group ID,
+home directory, default shell, and whether it is a built-in user that
+came with the %brand% installation. To reorder the list, click the
```
Comment (wblock) [inline userguide/account.rst]: Rearrange "To reorder the list" as "Clicking a column name will resort the list by that value."

File: userguide/account.rst [hunk 10]
```diff
@@ -168,46 +166,33 @@ operating system, as shown in
...
    the *root* account and changing this setting is highly discouraged.
 
+
 Except for the *root* user, the accounts that come with %brand%
 are system accounts. Each system account is used by a service and
 should not be used as a login account. For this reason, the default
 shell on system accounts is
 `nologin(8) <http://www.freebsd.org/cgi/man.cgi?query=nologin>`__.
-For security reasons, and to prevent breakage of system services, do
+For both security reasons and to prevent breakage of system services, do
```
Comment (wblock) [inline userguide/account.rst]: This reads like there are two security reasons.  The original was clearer, although the comma can be removed.

File: userguide/account.rst (hunk 2 above)
Comment (Mrt134) [reply userguide/account.rst]: I added bold to emphasise the values used by the convention, but I'll remove it.

File: userguide/account.rst (hunk 3 above)
Comment (Mrt134) [reply userguide/account.rst]: That one is part of the table - I can't find any whitespace errors apart from the usual table formatting.

**Pull request merged**